docs/Configuration/yaml-files.md

```diff
@@ -330,6 +329,8 @@ software:
   # path is relative to default.yml, teams/team-name.yml, or teams/no-team.yml
   app_store_apps:
     - app_store_id: '1091189122'
+  fleet_maintained_apps:
+    - fleet_maintained_app_name: Microsoft Excel
```

Comment:
@noahtalerman A few questions here:
1. I believe that strings with spaces need not have single quotes around them?
2. What does engineering have to do to make this happen? I'm curious how to dev note this for them.